packages/cli/src/modules/chat-hub/chat-hub-workflow.service.ts

```diff
@@ -471,11 +472,10 @@ export class ChatHubWorkflowService {
 	}
 
 	getSystemMessageMetadata(timeZone: string) {
-		const now = DateTime.now().setZone(timeZone).toISO({
-			includeOffset: true,
-		});
+		const now = inE2ETests ? DateTime.fromISO('2025-01-15T12:00:00.000Z') : DateTime.now();
```

Comment:
@Cadiac what do you think about including timestamp in request, together with timezone? That would allow us to set current time more flexibly in e2e. Especially for a test case of regeneration, [the bug below](https://github.com/n8n-io/n8n/pull/23396/changes#r2630356794) happened to make it possible to differentiate first and regenerate request, but otherwise two are identical and difficult to simulate two distinct requests with different response.